<!--
  1 failure:  :disappointed_rel...
  0 warning: 
  1 messages
  
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.3.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10255-VM7SFAA3TEZAZYRCWVQF3CQJJCXOK2B3YKUJ5FN36TUCDZRMFYIA
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/dk_titanium_mobile_PR-10255-VM7SFAA3TEZAZYRCWVQF3CQJJCXOK2B3YKUJ5FN36TUCDZRMFYIA/tests/Resources/ti.ui.textarea.addontest.js
  37:11  error  Missing space before value for key 'window'  key-spacing

/Users/build/jenkins/workspace/dk_titanium_mobile_PR-10255-VM7SFAA3TEZAZYRCWVQF3CQJJCXOK2B3YKUJ5FN36TUCDZRMFYIA/tests/Resources/ti.ui.textfield.addontest.js
  8:15  warning  'Titanium' is defined but never used  no-unused-vars

✖ 2 problems (1 error, 1 warning)
  1 error, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/iphone/Classes/TiUITextArea.m
+++ b/iphone/Classes/TiUITextArea.m
@@ -90,21 +90,17 @@
 {
   if (textWidgetView == nil) {
     TiUITextViewImpl *textViewImpl = [[TiUITextViewImpl alloc] initWithFrame:CGRectZero];
-    textViewImpl.delaysContentTouches = NO;
     [textViewImpl setTouchHandler:self];
+    textViewImpl.delaysContentTouches = NO;
     textViewImpl.delegate = self;
+    textViewImpl.text = @"";
+
     [self addSubview:textViewImpl];
     [textViewImpl setContentInset:UIEdgeInsetsZero];
     self.clipsToBounds = YES;
 
     lastSelectedRange.location = 0;
     lastSelectedRange.length = 0;
-    //Temporarily setting text to a blank space, to set the editable property [TIMOB-10295]
-    //This is a workaround for a Apple Bug.
-    textViewImpl.text = @" ";
-    textViewImpl.editable = YES;
-
-    textViewImpl.text = @""; //Setting TextArea text to empty string
 
     textWidgetView = textViewImpl;
   }

--- a/tests/Resources/ti.ui.textarea.addontest.js
+++ b/tests/Resources/ti.ui.textarea.addontest.js
@@ -1,0 +1,66 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti, Titanium */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+
+describe('Titanium.UI.TextArea', function () {
+	var win;
+
+	afterEach(function () {
+		if (win) {
+			win.close();
+		}
+		win = null;
+	});
+
+	// Tests adding and removing a TextArea's focus.
+	it('textArea in tabGroup', function (finish) {
+		var windowA, windowB, tabA, tabB, tabGroup;
+
+		this.timeout(7500);
+
+		windowA = Ti.UI.createWindow();
+		windowB = Ti.UI.createWindow();
+
+		tabA = Ti.UI.createTab({
+			window: windowA,
+			title: 'Tab A'
+		});
+
+		tabB = Ti.UI.createTab({
+			window:windowB,
+			title: 'Tab B'
+		});
+
+		tabGroup = Titanium.UI.createTabGroup({
+			tabs: [ tabA, tabB ]
+		});
+
+		windowA.addEventListener('open', function () {
+			var subwin, typingView, keyboardMessageView, keyboardMessage;
+
+			subwin = Ti.UI.createWindow({ backgroundColor: 'blur' });
+
+			subwin.addEventListener('open', function () {
+				finish();
+			});
+
+			typingView = Ti.UI.createView();
+			keyboardMessageView = Ti.UI.createView();
+			keyboardMessage = Ti.UI.createTextArea();
+
+			keyboardMessageView.add(keyboardMessage);
+			typingView.add(keyboardMessageView);
+			subwin.add(typingView);
+			tabA.open(subwin);
+		});
+
+		tabGroup.open();
+	});
+});

--- a/tests/Resources/ti.ui.textarea.addontest.js
+++ b/tests/Resources/ti.ui.textarea.addontest.js
@@ -34,7 +34,7 @@ describe('Titanium.UI.TextArea', function () {
 		});
 
 		tabB = Ti.UI.createTab({
-			window:windowB,
+			window: windowB,
 			title: 'Tab B'
 		});
 

--- a/tests/Resources/ti.ui.textfield.addontest.js
+++ b/tests/Resources/ti.ui.textfield.addontest.js
@@ -5,7 +5,7 @@
  * Please see the LICENSE included with this distribution for details.
  */
 /* eslint-env mocha */
-/* global Ti, Titanium */
+/* global Ti */
 /* eslint no-unused-expressions: "off" */
 'use strict';
 var should = require('./utilities/assertions');

